frontend/src/components/ActivityCard/ActivityCard.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import { Card, CardContent , CardMedia, Typography, CircularProgress } from '@material-ui/core';
+import './ActivityCard.css';
+
+class ActivityCard extends Component {
+  render() {
+    const { logo, title, tags, name, error, isLoading } = this.props;
+    let contentInfo;
+
+    if (isLoading) {
+      contentInfo = 'Loading data';
+    } else if (!error) {
+      contentInfo = <ul>
+        <li>Name: { name || '(not set)'}</li>
+        <li>Languages: { tags && tags.length ?
```

Comment:
Lets use *tags* rather then *languages* everywhere.

Reply:
in next request